diff --git a/mkosi b/mkosi
index 4b2e611..3753e0b 100755
--- a/mkosi
+++ b/mkosi
@@ -2757,7 +2757,7 @@ def make_tar(args: CommandLineArguments,
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
         run(["tar", "-C", os.path.join(workspace, "root"),
              "-c", "-J", "--xattrs", "--xattrs-include=*", "."],
-            env={"XZ_OPT": "-T0"},
+            env={"XZ_OPT": "-T0", "PATH": os.environ.get("PATH")},
             stdout=f, check=True)
 
     return f
