diff --git a/src/server-context-service.c b/src/server-context-service.c
index 3e1e7f1..ecd8392 100644
--- a/src/server-context-service.c
+++ b/src/server-context-service.c
@@ -89,7 +89,7 @@ on_notify_unmap (GObject    *object,
 static uint32_t
 calculate_height(int32_t width)
 {
-    uint32_t height = 180;
+    uint32_t height = 400;
     if (width < 360 && width > 0) {
         height = ((unsigned)width * 7 / 12); // to match 360×210
     } else if (width < 540) {
